### PR TITLE
A few minor IPC tweaks

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

IPCs now make different sounds when walking, can't wear underwear, and are no longer unable to speak encoded audio if they don't have a job

https://user-images.githubusercontent.com/93578146/222479012-fc98222a-325d-43a4-8079-57664697ae37.mp4

# Changelog

:cl:  
bugfix: fixes IPCs being unable to speak encoded audio language if they don't have a job 
tweak: IPCs make funny robot sounds when walking
tweak: IPCs can no longer wear underwear
tweak: changes ultra violence "grab" to be distinct from walk sound
/:cl:

### CODE REVIEW

**Comment by cuackles** [comment]
Based, can you post clips of the sounds ?

**Comment by Moggram** [comment]
>no longer unable to speak encoded audio if they don't have a job

**Comment by iamthesilly** [comment]
> Based, can you post clips of the sounds ?

done

File: code/datums/components/footstep.dm [hunk 1]
```diff
@@ -88,6 +88,9 @@
 			var/mob/living/carbon/human/H = LM
 			var/feetCover = (H.wear_suit && (H.wear_suit.body_parts_covered & FEET)) || (H.w_uniform && (H.w_uniform.body_parts_covered & FEET))
 			
+			if(H.dna.species.special_walk_sounds)
+				playsound(T, pick(H.dna.species.special_walk_sounds), 25, TRUE)
```
Comment (Bibby0110) [inline code/datums/components/footstep.dm]:
```suggestion
				playsound(T, pick(H.dna.species.special_walk_sounds), 25, TRUE)
				return
```

Comment (iamthesilly) [reply code/datums/components/footstep.dm]: Playing both sounds was intentional, but if you would prefer otherwise I can change that

**Comment by Moltijoe** [comment]
i think that sound is the same one used for the ipc martial art to register that a "grab" action has been landed, either change that one, or change this. them being the same sound could easily cause problems

File: code/datums/components/footstep.dm (hunk 1 above)
Comment (MajManatee) [reply code/datums/components/footstep.dm]: gee billy, mom gave you TWO walk sounds?

**Comment by Moltijoe** [comment]
please update the changelog that it's changing the ipc martial art grap action sound

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**